test/e2e/kubectl/kubectl.go

```diff
@@ -471,7 +470,7 @@ var _ = SIGDescribe("Kubectl client", func() {
 
 			ginkgo.By("Starting http_proxy")
 			var proxyLogs bytes.Buffer
-			testSrv := httptest.NewServer(utilnettesting.NewHTTPProxyHandler(ginkgo.GinkgoT(), func(req *http.Request) bool {
+			testSrv := httptest.NewServer(NewHTTPProxyHandler(ginkgo.GinkgoT(), func(req *http.Request) bool {
```

Comment:
[q] can [utilnettesting.NewHTTPProxyHandler](https://github.com/kubernetes/kubernetes/blob/69ab91a5c59617872c9f48737c64409a9dec2957/staging/src/k8s.io/apimachinery/pkg/util/httpstream/spdy/roundtripper_test.go#L339) in `staging/src/k8s.io/apimachinery/pkg/util/httpstream/spdy/roundtripper_test.go` be replaced as well?

Reply:
no need, we want to avoid the cross dependency of ginkgo, and ginkgo is only required for the e2e test